packages/plugin-loyalties-api/src/models/ScoreLogs.ts

```
@@ -138,17 +138,27 @@
             totalPointEarned: {
               $sum: {
                 $cond: {
-                  if: {$or: [{ $eq: ['$action', 'add'] }, { $gt: ['$changeScore', 0] }]},
-                  then: '$changeScore',
+                  if: {
+                    $or: [
+                      { $eq: ["$action", "add"] },
+                      { $gt: ["$changeScore", 0] },
+                    ],
+                  },
+                  then: "$changeScore",
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)